srcpkgs/mesa/template

```diff
@@ -78,6 +78,14 @@ case "$XBPS_TARGET_MACHINE" in
 		;;
 esac
 
+# Direct3D 12 for Hyper-V GPU-P / WSLg. This relies on additional
+# closed-source drivers which are only available on 64-bit glibc.
```

Comment:
these additional drivers are part of WSL?

Reply:
They are part of the GPU vendors' Windows drivers themselves and live in directories beneath `%SystemRoot%\System32\DriverStore\FileRepository`. WSL mounts this path at `/usr/lib/wsl/drivers` as a `9p` filesystem. `/usr/lib/wsl/lib` is an `overlayfs` with another glue library `libd3d12core.so` and in some cases additional vendor libs and tools, such as the CUDA libs and `nvidia-smi`.